F1WM/DatabaseModel/News.cs

```diff
@@ -23,5 +23,6 @@ public class News
 		public string Title { get; set; }
 		public string Subtitle { get; set; }
 		public string Text { get; set; }
-	}
+        public virtual F1NewsTopicmatch Topics { get; set; }
```

Comment:
The mapping is wrong here. A News may have multiple F1NewsTopicmatches, so the property should be an IEnumerable.